.github/pull_request_template.md

```diff
@@ -9,7 +9,6 @@
 * [ ] If applicable, have you updated the CHANGELOG?
 * [ ] Does your submission pass linting & tests? You can test on localhost using `./scripts/test`
 * [ ] Have you written new tests for your core features/changes, as applicable?
-* [ ] If this is a UI change, have you tested it across multiple browser platforms?
-* [ ] If this is a UI change, have you tested across multiple viewport sizes (ie. desktop versus mobile)?
+* [ ] If this is a UI change, have you tested it across multiple browser platforms on both desktop and mobile?
```

Comment:
Should this link to a short guide on how to do this or preferred platforms to test?

Reply:
I don't think so
The preferred platforms is just whatever you have available while hitting at least two of them to call it "multiple" (generally chromium + firefox on desktop, and app + default browser on mobile) and the testing is different on a per-pr basis